### PR TITLE
Add translation for signup page

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

**Review by daniel-beck — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

File: core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly [hunk 1]
```diff
@@ -188,15 +188,15 @@ THE SOFTWARE.
 
                                     const passwordStrength = function(score) {
                                         if (score > 80) {
-                                            return "Strong";
+                                            return "${%Strong}";
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly]: Didn't know it works in Javascript. Nice to know

Comment (LinuxSuRen) [reply core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly]: It's ok, I already test.

**Comment by oleg-nenashev** [comment]
Thanks @LinuxSuRen !

**Comment by LinuxSuRen** [comment]
My pleasure.

**Pull request merged**